cmake/Reprojucer.cmake

```diff
@@ -1938,6 +1988,12 @@ function(jucer_project_end)
     )
   endif()
 
+  if(JUCER_IOS)
+    _FRUT_create_xcassets_folder_from_icons(JUCER_XCASSETS)
+
```

Comment:
empty line leftover ?

Reply:
Addressed in https://github.com/McMartin/FRUT/pull/551/commits/74dc8d7e4588683dad9cc1c60d642d14d51f2d94#diff-d7ac1cfde9bf2d7ef9cccf6fab14bc6eR1990-R1991.